.github/workflows/publish-provena-interfaces.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     steps:
     - name: Check tag format
       run: |
-        if [[ ! $TAG_NAME =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ && ! $TAG_NAME =~ ^v[0-9]+\.[0-9]+$ && ! $TAG_NAME =~ ^v[0-9]+$ ]]; then
-          echo "Tag format is incorrect. Expecting vX.X.X or vX.X or Vx. Please create a new release with the correct tag format."
+        if [[ ! $TAG_NAME =~ ^v[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ && ! $TAG_NAME =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ && ! $TAG_NAME =~ ^v[0-9]+\.[0-9]+$ && ! $TAG_NAME =~ ^v[0-9]+$ ]]; then
+          echo "Tag format is incorrect. Expecting vX.X.X.X or vX.X.X or vX.X or vX. Please create a new release with the correct tag format."
```

Comment:
happy for this to be not included, but the 4th digits gives a good option to test pipelines without making a minor release until knowing pipelines are good.